aspnetcore/tutorials/min-web-api.md

```diff
@@ -352,4 +353,17 @@ Verify you can't post or get the secret field.
 
 See [Tutorial: Call an ASP.NET Core web API with JavaScript](xref:tutorials/web-api-javascript).
 
--->
\ No newline at end of file
+-->
+
+## Test API
+
+The following code shows the basic approach to testing minimal APIs:
+
+```csharp
+using Microsoft.AspNetCore.TestHost; 
```

Comment:
I don't think we want to show the test host. We want to show the [WebApplicationFactory](https://github.com/davidfowl/CommunityStandUpMinimalAPI/blob/davidfowl/preview7/TodoApi.Tests/TodoTests.cs)